tests/console/docker.pm

```diff
@@ -50,47 +50,62 @@ sub run {
     validate_script_output("docker search --no-trunc opensuse", sub { m/This project contains the stable releases of the openSUSE distribution/ });
 
     # images can be pulled from the Docker Hub
-    #   - pull minimalistic alpine image of declared version
+    #   - pull minimalistic alpine image of declared version using tag
     #   - https://store.docker.com/images/alpine
     my $alpine_image_version = '3.5';
     assert_script_run("docker pull alpine:$alpine_image_version", 300);
-    assert_script_run("docker images -q alpine:$alpine_image_version");
-    #   - pull hello-world image, typical docker demo image
+    assert_script_run("docker images alpine:$alpine_image_version | grep alpine");
+    #   - pull typical docker demo image without tag. Should be latest.
     #   - https://store.docker.com/images/hello-world
     assert_script_run("docker pull hello-world", 300);
-    assert_script_run("docker images -q hello-world:latest");
+    assert_script_run("docker images hello-world | grep latest");
```

Comment:
don't see how new check improve something. Can you state what exact false negatives you trying to solve ?

Reply:
Same answer for both ;)

Reply:
Agreed, you are right. JFYI I filed a bug at docker: https://github.com/docker/for-linux/issues/220